### DISCUSSION TITLE
Hubs Cloud Enterprise install missing OG:Image and proper sharing info

### DISCUSSION THREAD

**Discussion by normanwankowski** [mozilla/hubs#4784] [Hubs Cloud Questions] [answered]
When I am trying to share my enterprise install with custom url the OG: Image (site preview image) is not showing up. I checked my site https://tmvr.io with the Facebook lint tool.  https://developers.facebook.com/tools/debug/
Is there a way to assign this picture to get a pretty preview when sharing on social media?

**Comment by pattersonbl2** [comment]
I am not sure about this setup.   Have you asked anyone in the community discord server ?

**Comment by matthewbcool** [comment, marked as answer]
Try adding `<meta property="og:image" content="image_url" />` through the admin panel. 

![image](https://user-images.githubusercontent.com/4493657/142658816-ba356a85-df7f-46c8-ae23-dace758bac44.png)

**Comment by normanwankowski** [comment]
@matthewbcool yes this works! Thanks

Would be good to have this as part of the regular code.
So people can upload the Og: Image for the Hubs Cloud homepage via the admin backend.

![Bildschirmfoto 2021-11-19 um 18 08 41](https://user-images.githubusercontent.com/92306028/142663404-f5218760-4eca-4cb3-a779-d5fedb651c19.png)
